tests/testapp/test_fixedcharfield.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+from django.core.management import call_command
+from django.db import connection
+from django.test import TestCase, TransactionTestCase, override_settings
+
+from django_mysql.models import FixedCharField
+
+
+class TestFixedCharField(TestCase):
+    def test_invalid_length_type(self):
+        with pytest.raises(TypeError) as exc_info:
+            FixedCharField(length="4")  # type: ignore [arg-type]
+        assert "Expected integer value." in str(exc_info.value)
+
+    def test_invalid_length_too_short(self):
+        with pytest.raises(ValueError) as exc_info:
+            FixedCharField(length=-1)
+        assert "Length must be in the range" in str(exc_info.value)
+
+    def test_invalid_length_too_long(self):
+        with pytest.raises(ValueError) as exc_info:
+            FixedCharField(length=256)
+        assert "Length must be in the range" in str(exc_info.value)
+
+    def test_invalid_max_length(self):
+        with pytest.raises(TypeError) as exc_info:
+            FixedCharField(length=4, max_length=100)
+        assert '"max_length" is not a valid argument' in str(exc_info.value)
+
```

Comment:
Need tests that actually store data in the field and query it! otherwise there's no proof it works